### ISSUE TITLE
No model saved after running train.py

### ISSUE THREAD

**Issue by jpmcarrilho** [Joker316701882/Additive-Margin-Softmax#32]
After running train.py there is no model outputted by the pipeline.
Also, what kind of file you expect in the "--pretrained_model" flag, thanks!
